site/src/sphinx/setup-installation.rst

```diff
@@ -60,3 +60,12 @@ To stop the server, use the ``bin/shutdown`` script:
     You can also tail the log file::
 
         $ ./gradlew tail
+
+Run on docker
+--------------------
+You can also pull Central Dogma image from `Docker Hub <https://hub.docker.com/r/line/centraldogma>`
+and then run it on your docker:
+
+.. parsed-literal::
```

Comment:
- can omit .. parsed literal:: if you end the last sentence with :: i.e. on your Docker::
- docker -> Docker